Onboarding/README.md

```diff
@@ -3,40 +3,45 @@
 ![Onboarding Image](Figures/Logo.png "CanDIG Logo")
```

Comment:
I am a bit confused by this file. It has a lot of changes for indentation. Is that on purpose? Just curious.